### PR TITLE
Adds the --keep-app-data-directory command line argument for running tests on Android browsers

### PR DESCRIPTION
**Pull request by rmhasan**
This new argument will make ChromeDriver not delete the app data directory when launching an Android browser. This CL also creates an android specific command line argument group and moves several arguments into that group

Bug: crbug.com/1233679, crbug.com/1261285

### CODE REVIEW

**Comment by rmhasan** [comment]
friendly ping

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
The failure here is because of https://github.com/web-platform-tests/wpt/pull/31418, I'll try to close and reopen to see if it goes away.

**Comment by foolip** [comment]
Now the documentation job is failing. @jgraham you were looking at Sphinx stuff recently, did you change something, or was it already broken? Ideas for how to work around it?

**Comment by rmhasan** [comment]
Does anyone know why the mozilla:gecko-blocked tag was added to this PR?

**Comment by foolip** [comment]
@rmhasan I think that's part of how Gecko's two-way sync works. I'm not sure exactly what it means, but am quite sure it can be ignored, it doesn't block the PR in any way.

**Comment by foolip** [comment]
I'll try to close and reopen to see if the documentation failure has been fixed.

**Comment by rmhasan** [comment]
I ran the command for the documentation test, "./tools/ci/website_build.sh" and it outputs "Could not find a version that satisfies the requirement sphinx==2.4.4...". I don't think this failure is caused by this PR. Can we merge this change? I am currently blocked on this. I need this command line arg so that I can create a test that runs an Android browser after putting some files in the app data directory. I can't merge it myself since I don't have write access to the repository.

**Comment by foolip** [comment]
@rmhasan I agree the problem wasn't caused by this PR. Thanks @WeizhongX for merging.

**Pull request merged**
